_events/2019-12-07-djangogirls.yml

```
@@ -0,0 +1,19 @@
+title: "Django Girls São Paulo 4ª Edição"
```

```suggestion
---
title: "Django Girls São Paulo 4ª Edição"
```

Coisa do yml, se não ele n compila o evento 😞